### ISSUE TITLE
Circle and color

### ISSUE THREAD

**Issue by starryskystar** [SkalskiP/make-sense#233]
I am adding the function of drawing circles and different colors for each line based on your project, as required by the scene. Any good suggestions?

**Comment by starryskystar** [comment]
Looking forward to your reply  🙏

**Comment by SkalskiP** [comment]
Hi, @starryskystar sorry for the late response. I had some problems that slowed me down. Did you started to work on that issue?

**Comment by starryskystar** [comment]
Year, our team needs these features

**Comment by SkalskiP** [comment]
🥳 Is there any initial PR that I could take a look at? Any way that I could assist you?

**Comment by SkalskiP** [comment]
I also assume this is a duplicate of: https://github.com/SkalskiP/make-sense/issues/218. @starryskystar am I correct?

**Comment by starryskystar** [comment]
Yes, no problem at all. But I have a few small personal needs 🤗  @

**Comment by starryskystar** [comment]
Thank you for your reply 🥳

**Comment by SkalskiP** [comment]
> Yes, no problem at all. But I have a few small personal needs 🤗

@starryskystar, can I help somehow?

**Comment by starryskystar** [comment]
🤠 Thank you, I have some ideas! If I have a problem, I will ask you for help!

**Comment by SkalskiP** [comment]
Sure, just let me know ;)

**Comment by SkalskiP** [comment]
Hi @starryskystar! Any updates here? :)

**Comment by AstralBit** [comment]
No one is developing this function now, if not, I will implement this thing

**Comment by SkalskiP** [comment]
Go for it! 🚀
